lib/roo/xls/excel_2003_xml.rb

```diff
@@ -18,8 +18,10 @@ def initialize(filename, options = {})
       unless File.file?(@filename)
         fail IOError, "file #{@filename} does not exist"
       end
-      @doc = load_xml(@filename)
+      @doc = ::Roo::Utils.load_xml(@filename)
     end
+    namespace = @doc.namespaces.select{|xmlns, urn| urn == 'urn:schemas-microsoft-com:office:spreadsheet'}.keys.last
+    @namespace = namespace.nil? || namespace.empty? ? 'ss' : namespace.split(':').last
```

Comment:
What about using ```key``` instead of ```select{...}.keys.last```
http://ruby-doc.org/core-1.9.3/Hash.html#method-i-key

    namespace = @doc.namespaces.key('urn:schemas-microsoft-com:office:spreadsheet') || 'ss'
    @namespace =  namespace.split(':').last